src/frontend/komponenter/Fagsak/Søknad/BarnMedOpplysninger.tsx

```diff
@@ -15,8 +16,8 @@ const BarnMedOpplysninger: React.FunctionComponent<IProps> = ({ barn }) => {
         : 'ukjent';
 
     return (
-        <div>
-            <Checkbox
+        <div className={'søknad__panel--gruppebarn'}>
+            <CheckboxLesbar
```

Comment:
Jeg ser for meg at vi egentlig kunne trukket ut disse komponentene til familie-felles-frontend. Egentlig kunne vi kanskje kalt de <Familie*** /> . Hva tenker du om det?

Eneste vi måtte endret på da er å endre tilbake til at komponentene kan ta inn en lesevisning prop 🤦 😆

Reply:
Jeg er enig, har endra navn nå :) 
Når det kommer til å trekke det ut i familie-felles-frontend lager jeg et [favrokort](https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=Tea-1072) på det, så jeg kan begynne å se på prodfeilen. Høres det greit ut?

Reply:
Det høres veldig bra ut 👍